src/python/pants/option/global_options.py

```diff
@@ -1202,6 +1205,13 @@ def register_bootstrap_options(cls, register):
             default=DEFAULT_EXECUTION_OPTIONS.remote_store_rpc_concurrency,
             help="The number of concurrent requests allowed to the remote store service.",
         )
+        register(
+            "--remote-store-batch-api-size-limit",
+            type=int,
+            advanced=True,
+            default=DEFAULT_EXECUTION_OPTIONS.remote_store_batch_api_size_limit,
+            help="The total size of blobs allowed to be sent in a single batch API call to the remote store.",
```

Comment:
```suggestion
            help="The maximum total size of blobs allowed to be sent in a single batch API call to the remote store.",
```

Reply:
Done (and updated similar help text in brfs and fs_util.)